src/coreclr/hosts/coreshim/CoreShim.cpp

```diff
@@ -0,0 +1,275 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
cc @morganbr This is the library I am adding to enable hosting when starting in a native entry point. I have also written a mechanism that allows native EXEs to be launched as the test entry point.

Reply:
This is equivalent to `corerun.exe` - my understanding is that all the projects under the `hosts/` are for testing and all real hosts should be contained in the core-setup repo. Am I missing the point of this directory?